react/features/conference/components/native/carmode/Conference.tsx

```diff
@@ -66,15 +81,6 @@ const CarmodeTab = (): JSX.Element => {
                 style = { styles.microphoneContainer }>
                 <MicrophoneButton />
             </View>
-            <View
```

Comment:
Why this change?

Reply:
Because he have the footerComponent  that we can use.

Reply:
It doesn't, you had to disable the linter!

Reply:
No longer. Please check the latest commits.